src/app/footer/footer.component.scss

```diff
@@ -0,0 +1,19 @@
+@import '../../styles/theme';
+@import '../../styles/_sizes';
+
+#stlFooter {
+  min-height: 4em;
```

Comment:
To do a sticky footer I think you'll need a fixed-height footer.

Reply:
This code works.  The footer height even changes as the menu goes vertical for mobile sizes.